Tests/SwiftDocCTests/Diagnostics/DiagnosticConsoleWriterDefaultFormattingTest.swift

```diff
@@ -245,6 +245,35 @@ class DiagnosticConsoleWriterDefaultFormattingTest: XCTestCase {
         """)
     }
 
+    func testDisplaysSource_WithEmojis_ProperlyHighlightsSource() {
+        let identifier = "org.swift.docc.test-identifier"
+        let summary = "Test diagnostic summary"
+        let explanation = "Test diagnostic explanation."
+        let baseURL =  Bundle.module.url(
```

Comment:
For a behavior that's this isolated I don't think we need to add new files to TestBundle.docc which is used by many different tests. 

The existing TestBundle.docc/TestTutorial.tututorial file has a "💻" if this test only needs one emoji to reproduce. Alternatively you could make a multi-line string literal and write it to a temp file if this needs more specific output.

Reply:
Great point! I wasn't sure about adding this file either, thank you for the suggestion. I have pushed a commit fixing this to use the existing `TestBundle.docc/TestTutorial.tutorial` file.